trunk/NDHMS/Data_Rec/rt_include.inc

```diff
@@ -140,6 +144,24 @@
   REAL,  DIMENSION(50)     :: BOTWID,TOPWID,HLINK_INIT,CHAN_SS,CHMann !Channel parms from table
   REAL,  DIMENSION(50)     :: TOPWIDCC, NCC  !topwidth and mannings n of compund
 
+  ! VARIABLES FOR  RESERVOIRS
+  REAL :: water_elevation
+  REAL :: routing_period
+  REAL :: previous_timestep_inflow
+  REAL :: inflow
+  REAL :: previous_day_inflow
+  REAL :: outflow
+  REAL :: lateral_inflow
+  REAL :: lake_area
+  REAL :: weir_elevation
+  REAL :: weir_coeffecient
+  REAL :: weir_length
+  REAL :: orifice_elevation
+  REAL :: orifice_coffecient
+  REAL :: orifice_area
+  REAL :: max_depth
+  INTEGER :: lake_number
+
   REAL, allocatable, DIMENSION(:)      :: RESHT  !reservoir height
```

Comment:
This one is still necessary, which is a bit puzzling to me.

Reply:
at least compiling fails if I take it out. It may make sense, it's just not obvious to me that this should still be here after this refactor.

Reply:
I think you mean just the resht here?

Reply:
yes

Reply:
Resolved by comment